Pdf enhancements

dan-palmer opened the pull request:
Padding on the subsections, fix the page size(reduce the zoom effect), change the pdf testing approach.

Commit:
Remove unused styles

Commit:
Make use of the print styles

This sets the document size removing the zoomed in effect.

Commit:
Increase left padding on risk subsections

Makes the padding more obvious!

Commit:
Add a matcher for validating pdf contents

In the case of the text not being the same you get a pretty lil diff.

Failures:

  1) printing a PER when a PER is completed with all answers as no user prints the PER
     Failure/Error:
       expect(page.body).
         to validate_as_pdf_that_contains_text(‘some-file.txt’)

       expected pdf to contain the same text as in: some-file.txt

       the following snippet will show the diff:


       -10 Dec 1970     46      M       British   Photo unavailable
       +10 Dec 1970     46      M       American   Photo unavailable

Commit:
Use pdf text matcher in printing PER spec

The way we compared text in the document made it hard to find the cause
of the issues when they occurred. Having a document that kind of
represents the pdf in terms of layout makes it easier to resolve issues
visually.

lostie commented on app/assets/stylesheets/print.scss:
```diff
@@ -319,99 +318,6 @@ table.vehicles {
   }
 }
 
-table.detainee {
```
🌈 👯 🌈

lostie commented on spec/features/printing_spec.rb:
```diff
@@ -115,89 +81,13 @@
       })
     }
 
-    let(:front_cover_alerts){
```
![image](https://cloud.githubusercontent.com/assets/67125/24258464/d02e357a-0fe5-11e7-97ad-2a5c3c0d23d2.png)

Commit:
Fix photo unavailable text alignment in PDF

Commit:
Use pdftotext to extract text from pdf under test

The algorithm pdf-reader uses to detect lines of text has some
shortcomings. The output can appear with extraneous whitespace.
Pdftotext produces better output so we’ve switched to that.

This method of testing isn’t the greatest, layout changes will alter
the test results. This has been useful in flagging some issues such
as the different fonts used in the different environments(OSX vs linux).
It might be better to test the HTML we pass to webkit html to pdf.

lostie reviewed (approved):
LGTM 👍
